### PR TITLE
Add ce diagrams to README.md (INF-1118)

### PR DESCRIPTION
**Pull request by rc-jacob**
Logical diagrams based off those found in Brian Lin's office, all made with mermaid-js

### CODE REVIEW

**Review by ColeBollig — CHANGES_REQUESTED** [review]
Just some minor nitpicky changes. I can't speak for the full validity of the graph flows and will yield the accuracy of these graphs to @brianhlin

File: README.md [hunk 1]
```diff
@@ -43,3 +43,92 @@ Please share any issues or questions regarding the HTCondor-CE via the following
...
+%%   are labelled by double letters (AA - ZZ)
+%% - All subgraph names are capitalized (Blahp)
+%%   while nodes with the same name are lowercase (blahp)
+
+subgraph HTCondor-CE
+	%%direction LR %% Flowchart direction statement overrides statements in connected subgraphs; comment these out.
+	A[[SchedD]] --> B(Job Router)
+	B -. Routed Job .-> A
+	A -.-> C{Grid Mgr.}
+	D[(Log)] --> C
```
[the remark this answers is not in the document]
Comment (rc-jacob) [reply README.md:66]: This seems like a better idea. I'll add this suggestion.

**Comment by brianhlin** [comment]
We merged this into the new main branch in #595 so we don't need this particular PR anymore. @rc-jacob we were able to retain your specific commits, though!

**Comment by timtheisen** [comment]
Superseded by #595

**Pull request closed without merging**